stage0/run.go

```diff
@@ -165,20 +159,78 @@ func Prepare(cfg PrepareConfig, dir string, uuid *types.UUID) error {
 
 	cdoc, err := json.Marshal(cm)
```

Comment:
nit, while you're in here, can you do a global s/cm/pm/ and s/cdoc/pmb/ (or similar)?